PBAB-QUICK-START.md

```diff
@@ -0,0 +1,26 @@
+# Gain Access to Twitter and Discord APIs
+
+1. Discord - https://discord.com/developers/docs/resources/webhook
+2. Twitter - https://developer.twitter.com/en/docs/twitter-api
+
+# Fork 
+
+Fork this repo into your organization.
+
+# Adjust Code. 
+
+1. Allow your contracts here - https://github.com/ArtBlocks/alertbot/blob/main/src/index.ts#L13-L15, remove ab contracts. 
+2. Update this endpoint https://github.com/ArtBlocks/alertbot/blob/main/src/api_data.ts#L46 - use {contractAddress/tokenId} 
+3. Provide location for smaller images. If none available, use your defualt image bucket. https://github.com/ArtBlocks/alertbot/blob/main/src/config.ts#L20. 
+      - NOTE: If default image bucket is used, you may run up against twitter image size limits.
+5. Set rest of environment https://github.com/ArtBlocks/alertbot/blob/main/.env.example. When ready, we can provide the webhook secret. 
```

Comment:
@Jbern16 Is this webhook secret something that we can issue on a per-partner basis (and revoke if needed due to access breech or something like that)?

Reply:
Will have to sync with @yoshiwarab to see best way to go about this

Reply:
That being said, I think we can figure this out when a partner gets to this point and shouldn't hold up oboarding interested parties. 

@yoshiwarab thoughts?

Reply:
> That being said, I think we can figure this out when a partner gets to this point and shouldn't hold up oboarding interested parties.

Yep yep, I agree, doesn't need to be a blocker for merging this and starting to help interested partners get moving on onboarding. That being said PLEASE PLEASE PLEASE lets not share a webhook secret that is universal across all partners, especially if it is also shared with us as a partner.

Reply:
yes of course